web/sites/default/config/group.role.elected_official-member.yml

```diff
@@ -12,41 +12,7 @@ audience: member
 group_type: elected_official
 permissions_ui: true
 permissions:
-  - 'access group_media overview'
-  - 'access group_node overview'
```

Comment:
Bureau member has "access group_node overview" and  "access group_media overview". Are they removed for Elected member on purpose?

Reply:
Nice catch. Fixed in the next commit.